mocked_servers/https/Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM golang:1.17 as build
+FROM golang:1.17 AS build
```

Comment:
Is this needed or is this just to match convention?

Reply:
This is not needed. I just copied over that part of the Dockerfile from the other mocked_servers. Didn't notice they were different.